.github/workflows/kubernetes_test.yaml

```diff
@@ -32,9 +32,9 @@ jobs:
       run:
         shell: bash -l {0}
     steps:
-      - name: 'QHUB_GH_BRANCH set for PR'
```

Comment:
Since this is tied - we might need better tags for our Docker images.
- Up to now they have been coupled with the code so they have been following  semver
- Using the branch as part of a tag is good practice
- I would suggest having a combination of `branch-shortsha-date` to ensure reproducibility and easier tracking plus we will eventually move to CalVer for nebari so we can remove the need for a SemVer tag

Reply:
Sounds good to me! As discussed in yesterday's meeting, we will use `date-branch-shortSha`. This will need to be updated in the nebari-docker-images repo.